packages/salesforcedx-vscode-core/package.json

```diff
@@ -976,6 +976,11 @@
           "default": false,
           "description": "%conflict_detection_enabled_description%"
         },
+        "salesforcedx-vscode-core.disableSourceTrackingForDeployAndRetrieve": {
+          "type": "boolean",
+          "default": false,
```

Comment:
Change this to true

Reply:
The default setting should be false only. We are providing an option to enable source tracking if you want.

Reply:
This one we may want to re-confirm before merging in